lib/diagnostic.js

```diff
@@ -5,17 +5,17 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (previous, current) {
-
+const min = function (a, cV, cI, array) {
```

Comment:
This will totally work, but functions passed to a `reduce` should only have 2 arguments